src/content/reference/react-dom/hydrate.md

```diff
@@ -192,10 +192,10 @@ export default function App() {
 
 </Sandpack>
 
-This way the initial render pass will render the same content as the server, avoiding mismatches, but an additional pass will happen synchronously right after hydration.
+Dengan cara ini, proses render awal akan me-*render* konten yang sama dengan server, menghindari ketidakcocokan, tetapi ada tambahan proses synchronously setelah hydration.
 
 <Pitfall>
 
-This approach makes hydration slower because your components have to render twice. Be mindful of the user experience on slow connections. The JavaScript code may load significantly later than the initial HTML render, so rendering a different UI immediately after hydration may feel jarring to the user.
+Metode ini membuat hydration lebih lambat karena komponen Anda harus di-*render* dua kali. Pertimbangkan pengalaman pengguna pada koneksi lambat. Kode JavaScript dapat dimuat jauh setelah render HTML awal, sehingga me-*render* UI yang berbeda langsung setelah hydrasi dapat terasa tidak nyaman bagi pengguna..
```

Comment:
Same with https://github.com/reactjs/id.react.dev/pull/528#discussion_r1181541016 but in word "hydration"

Reply:
I have corrected the typo

Reply:
don't forget to "resolve" when you have been changes!